src/components/i18nStrings.vue

```diff
@@ -135,11 +136,26 @@ export default {
 					this.$store.commit('cleanStateFor', this.path)
 				}
 			},
-      		deep: true
+			deep: true,
+			
+			"$store.state.languages": function(languages) {
+				for (const lang in languages) {
+					if (languages[lang]) {
+						this.addPair(lang)
+					}
+				}
+			},
 		},
 	},
 	created() {
 		this.state = this.value
+		const languages = this.$store.state.languages
```

Comment:
duplicate functionality here

Reply:
The one we have mentioned in created will get executed at the time of component gets created. However there are other components in other pages which are expecting to have their section also gets populated whenever the language is updated in store.

Reply:
it is a very bad pattern to copy-paste code around the places. this must be changed.

Reply:
extracted functionality to a method